src/NUnit3TestAdapterInstallV2/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="NUnit.NUnit3TestAdapter" Version="4.0.0.0" Language="en-US" Publisher="Terje Sandstrom" />
+    <Identity Id="NUnit.NUnit3TestAdapter" Version="3.6.0.1" Language="en-US" Publisher="Terje Sandstrom" />
```

Comment:
About the nuget package - this change is really only relevant to the vsix, but there are 2 other changes we get along, 253 and 254, and that means the nuget should be updated too. 
 I have tried using the nuget 3.5.1 package on VS 2017, and it works as it should, so there is nothing more needed to be done there.   We need to update the text for it however.